Lesson16/batikan/contactBook.js

```diff
@@ -0,0 +1,240 @@
+/*
+===========================================================
+  SIMPLE CONTACT BOOK APPLICATION
+===========================================================
+In this project, you'll create a Contact Book to store and
+manage basic info: name, phone, and email.
+
+You'll practice:
+1. Arrays and objects
+2. Loops (for, for-of, findIndex, etc.)
+3. Conditionals (if-else)
+4. Basic CRUD (Create, Read, Update, Delete) functionality
+
+Below is a step-by-step guide, with sample code and 
+comments explaining what each section does. Run this file
+in Node.js or in your browser's console to see the output.
+*/
+
+/*
+-----------------------------------------------------------
+  STEP 1: Setup and Initial Data
+-----------------------------------------------------------
+1. Create an array named 'contacts' with a few initial 
+   sample contacts.
+2. Each contact is an object with 'name', 'phone', and 
+   'email' properties.
+*/
+
+const contacts = [
+  { name: 'Alice', phone: '123-456-7890', email: 'alice@example.com' },
+  { name: 'John', phone: '123-456-7890', email: 'john@example.com' },
+  { name: 'Janna', phone: '333-323-3222', email: 'alice@yahoo.com'},
+];
+
+console.log(contacts[0].name);
+
+/*
+-----------------------------------------------------------
+  STEP 2: Display All Contacts
+-----------------------------------------------------------
+Function: displayAllContacts()
+- Loops over the 'contacts' array.
+- Logs a descriptive string for each contact.
+
+Example output:
+  Name: Alice, Phone: 123-456-7890, Email: alice@example.com
+  -----
+*/
+function displayAllContacts() {
+  for (let i = 0; i < contacts.length; i++) {
+    console.log(
+      `Name: ${contacts[i].name}, Phone: ${contacts[i].phone}, Email: ${contacts[i].email}`,
+    );
+    console.log('-----');
+  }
+}
+
+console.log('Initial contact list:');
+displayAllContacts();
+
+
+/* contactFinder helper function */
+
+  const contactFinder = function findContact (name){
+    for (let i = 0; i < contacts.length; i++){
+      if (name && contacts[i].name.includes(name)
```

Comment:
Name is a string, so you need to use comparison (===) instead of the includes function.
```suggestion
      if (name && contacts[i].name === name
```